titan/assetlist.json

```diff
@@ -281,6 +281,120 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
         }
       ]
+    },
+    {
+      "description": "SOL on Titan. Bridge into titan chain from Solana to Titan through Titan Power flow bridge https://powerflow.asia/bridge",
+      "denom_units": [
+        {
+          "denom": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/sol",
+          "exponent": 0,
+          "aliases": [
+            "lamport"
+          ]
+        },
+        {
+          "denom": "sol",
+          "exponent": 9
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/sol",
+      "name": "SOL",
+      "display": "sol",
+      "symbol": "SOL",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "solana",
+            "base_denom": "So11111111111111111111111111111111111111112",
+            "contract": "3FCYJzQBTfZg8rbCpbVEvRPtJsGbhGzfQmJMAkcQ7bi6"
+          },
+          "provider": "powerflow.asia, titanlab.io"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titan/images/sol.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titan/images/sol.png"
+        }
+      ]
+    },
+    {
+      "description": "Meow on Titan. Bridge into titan chain from Solana to Titan through Titan Power flow bridge https://powerflow.asia/bridge",
+      "denom_units": [
+        {
+          "denom": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/meow",
+          "exponent": 0
+        },
+        {
+          "denom": "meow",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/meow",
+      "name": "MEOW",
+      "display": "meow",
+      "symbol": "MEOW",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "solana",
+            "base_denom": "BUhS5coXEt9hcxN3JSpGYUWSKbNo96RsKu52LcMo12rf",
```

Comment:
Oh, the verification script is currently failing because these assets aren't registered under the Solana assetlist yet (which I realize now is where the images really should be uploaded)